packages/query-engine/source/utils/errors/BadRequestError.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response, Request } from "express";
+import { constants } from "@hypertool/common";
+
+const { httpStatuses } = constants;
+
+const BadRequestError = (
+    error: any,
+    request: Request,
+    response: Response,
+    next: NextFunction,
+): void => {
+    if (error.code === "ER_BAD_HOST_ERROR") {
+        response.status(httpStatuses.BAD_REQUEST).json({
+            error: "Bad Host Request",
```

Comment:
Error messages should be proper sentences. That's the convention we follow everywhere else.

Reply:
The messages are full sentences here. The `error` field is the name of the error.